01_Data/src/layers/sequelize/model/TransactionEvent/TransactionEvent.ts

```diff
@@ -53,10 +53,10 @@ export class TransactionEvent extends Model implements OCPP2_0_1.TransactionEven
   declare transactionDatabaseId?: string;
 
   @BelongsTo(() => Transaction)
-  declare transaction?: OCPP2_0_1.TransactionType;
+  declare transaction?: Transaction;
 
   @Column(DataType.JSON)
-  declare transactionInfo: OCPP2_0_1.TransactionType;
+  declare transactionInfo: Transaction;
```

Comment:
Can we reset it as `OCPP2_0_1.TransactionType`? Or I think we can just remove this field because `TransactionEvent` no longer implements `OCPP2_0_1.TransactionEventRequest` and then nothing force us to have this field. All info in `transactionInfo` is stored in `Transaction` table.